recipes/sqlite3/all/conanfile.py

```diff
@@ -131,13 +170,15 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package_multi"] = "SQLite"
         self.cpp_info.components["sqlite"].names["cmake_find_package"] = "SQLite3"
         self.cpp_info.components["sqlite"].names["cmake_find_package_multi"] = "SQLite3"
+        self.cpp_info.components["sqlite"].builddirs.append(self._module_subfolder)
+        self.cpp_info.components["sqlite"].build_modules["cmake_find_package"] = [self._module_file_rel_path]
```

Comment:
No multi support?

Reply:
I don't think that `cmake_find_package_multi` defines the variables which are aliased.
Anyway, you should never rely on `cmake_find_package_multi` for libraries with built-in Find module files (like sqlite3, zlib, libcurl, openssl, bzip2, libjpeg), if you call find_package without CONFIG keyword.